lib/dataTrigger/DataTriggerExecutionResult.js

```diff
@@ -0,0 +1,32 @@
+class DataTriggerExecutionResult {
+  /**
+   * @param {DataTriggerExecutionError[]} errors
+   */
+  constructor(errors = []) {
+    this.errors = errors;
+  }
+
+  /**
+   * @param {DataTriggerExecutionError} error
+   */
+  addError(...error) {
+    this.errors.push(...error);
+  }
+
+  /**
+   * Get all data trigger execution errors
+   * @return {DataTriggerExecutionError[]}
+   */
+  getErrors() {
+    return this.errors;
+  }
+
+  /**
+   * @return {boolean}
+   */
+  isOk() {
+    return !this.errors.length;
```

Comment:
Strict type comparison always better: `this.errors.length === 0`

Reply:
:+1: